packages/grid/x-data-grid-premium/src/DataGridPremium/useDataGridPremiumComponent.tsx

```diff
@@ -87,91 +89,90 @@ export const useDataGridPremiumComponent = (
   inputApiRef: React.MutableRefObject<GridApiPremium> | undefined,
   props: DataGridPremiumProcessedProps,
 ) => {
-  const privateApiRef = useGridInitialization<GridPrivateApiPremium, GridApiPremium>(
-    inputApiRef,
-    props,
-  );
+  const apiRef = useGridInitialization<GridPrivateApiPremium, GridApiPremium>(inputApiRef, props);
```

Comment:
`s/privateApiRef/apiRef/` because our community/pro/premium files were inconsistent with the naming of that.

Beside that substitution, I've added to pro & premium the new   `virtualizationStateInitializer` and `useGridVirtualization` hooks.